packages/react-client/src/types.ts

```diff
@@ -169,9 +169,8 @@ export type ScreenshareState = {
 } | null;
 
 export type Device = {
-  streamedTrack: MediaStreamTrack | null;
-  streamedTrackId: TrackId | null;
-  stream: MediaStream | null;
+  currentlyStreamed: { track: MediaStreamTrack; trackId: TrackId; stream: MediaStream } | null;
+  rawStream: MediaStream | null;
```

Comment:
I'm not sure if `currentlyStreamed` is the right name. It's descriptive but a little bit odd. Since it's our public API, the name will stick with us. My suggestion is to use `participantStream` or `broadcastStream` instead. @mironiasty WDYT?

Reply:
`broadcastStream.stream` and `broadcastStream.track`, or `broadcast.stream` and `broadcast.track`?

Reply:
duh, looks like I didn't publish my previous comment before

Reply:
To remove `rawStream` and `currentlyStreamed` we could implement one stream (with middleware) and additional field as an indicator:
https://linear.app/swmansion/issue/FCE-450/expose-local-stream-with-middleware

Reply:
> To remove `rawStream` and `currentlyStreamed` we could implement one stream (with middleware)
I will take time to try to achieve that in the simplest way.